public/diagnostic.js

```diff
@@ -23,3 +23,10 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Order = Ember.Object.extend({
```

Comment:
I think you're overthinking this a bit! You didn't have to get the data from store here, you can just create the object. 
```js
const Order = Ember.Object.extend({
  price: this.price,
  quantity: this.quantity,
  orderPrice: Ember.computed('price', 'quantity', function() {
    return `${this.get('price')}` * `{this.get('quantity')}`;
  })
});
```